frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -580,6 +588,30 @@ export function ActionFilterRow({
                                                             </>
                                                         ),
                                                     },
+                                                    {
+                                                        label: () => (
+                                                            <>
+                                                                {index > 0 && (
+                                                                    <>
+                                                                        <div className="px-2 py-1">
+                                                                            <LemonCheckbox
+                                                                                checked={!!filter.optionalInFunnel}
+                                                                                onChange={(checked) => {
+                                                                                    updateFilterOptional({
+                                                                                        ...filter,
+                                                                                        optionalInFunnel: checked,
+                                                                                        index,
+                                                                                    })
+                                                                                }}
+                                                                                label="Optional step"
```

Comment:
I think it's worth adding a tooltip here / link to docs because it's not intuitive to me what happens around drop off calcs when this is checked.